new-log-viewer/src/components/Layout.css

```diff
@@ -0,0 +1,52 @@
+
+
+.layout {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+}
+
+.menu-bar {
```

Comment:
Shall we move these to their dedicated stylesheets? e.g. `src/components/MenuBar/index.css`?